src/btree/bt_walk.c

```diff
@@ -234,7 +234,7 @@ __split_prev_race(WT_SESSION_IMPL *session, WT_REF *ref, WT_PAGE_INDEX **pindexp
 
 /*
  * __tree_walk_internal --
- *     Move to the next/previous page in the tree.
```

Comment:
I would improve this comment a bit, I like what you're going for but its pretty opaque. "Move to the next/previous page in the tree, skipping pages in WT_REF_DELETED state and for other reasons. Those reasons are generally controlled by the flags passed in to this function."

Reply:
Thanks, fixed in fdacaf4.